### PR TITLE
feat: add DISABLE_DEBUG environment variable

### PR DESCRIPTION
**Pull request by fmvilas**
#172

### CODE REVIEW

**Comment by fmvilas** [comment]
@magicmatatjahu one more feature to mimic in Studio 😂

**Review by derberg — COMMENTED** [review]
how about the package.json stuff I suggested in the issue

also, a thought: if now by default we have logs, shouldn't we keep it this way, and only disable on request? to not break any functionality for folks that use playground on their own? so basically? it should be `true` by default or called `DISABLE_DEBUG` and `false` by default

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Editor for AsyncAPI documentation.",
   "scripts": {
     "start": "node src/server/index.js",
+    "dev": "node src/server/index.js",
```
Comment (fmvilas) [inline package.json:7]: @derberg Added your suggestion but now `start` and `dev` look the same. Do you think it's still worth it to have `npm run dev`?

Comment (derberg) [reply package.json:7]: I guess dev is not really needed

**Review by derberg — APPROVED** [review]
just update PR title and :shipit:

**Pull request merged**